client/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text>Welcome to the Gym App</Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Can this be put into its on seperate file "Styles" and have a proper name, e,g containerStyle. I see this being used in a few different views

Reply:
Yeah alright. Actually I'll just remove it for now... it was just part of the template.